src/components/Onboarding/Onboarding.js

```diff
@@ -38,7 +53,7 @@ function Onboarding(props: Props) {
 
   return (
     <ModalDialog
-      className={props.className}
+      className={`${props.className}`}
```

Comment:
-1: Why the \`\`?

Reply:
I had another class in there in between - will remove.

Reply:
Fixed